style.css

```
@@ -2135,6 +2135,162 @@ body[data-bg="wallpaper"] .micIcon {
+
+.imageSearchPopover {
+	width: 300px;
+	padding: 16px;
+	border-radius: 22px;
+	box-shadow: 0 12px 32px rgba(0, 0, 0, 0.22);
+	z-index: 1000;
+	box-sizing: border-box;
+	animation: imageSearchPopoverIn 0.18s ease-out;
+}
+
+@keyframes imageSearchPopoverIn {
```

Stylelint (17.9.0)

[error] Expected keyframe name "imageSearchPopoverIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)